themes/hello-friend-ng/layouts/partials/resources.html

```diff
@@ -3,106 +3,211 @@
     <div class="row">
```

Comment:
This file was edited by Prettier (I didn't change any of the code, just formatting) but if this is an issue, I can revert this change.